src/D2L.Bmx/Okta/OktaClient.cs

```diff
@@ -187,6 +188,19 @@ async Task<OktaApp[]> IOktaAuthenticatedClient.GetAwsAccountAppsAsync() {
 				?? throw new BmxException( "Error retrieving AWS accounts from Okta." );
 	}
 
+	async Task<OktaSession> IOktaAuthenticatedClient.GetCurrentOktaSessionAsync() {
+		OktaSession? session;
+		try {
+			session = await httpClient.GetFromJsonAsync(
+				"sessions/me",
+				JsonCamelCaseContext.Default.OktaSession );
+		} catch( Exception ex ) {
+			throw new BmxException( "Request to retrieve session expiry from Okta failed.", ex );
```

Comment:
```suggestion
			throw new BmxException( "Request to retrieve session from Okta failed.", ex );
```
Similarly below.
Also consider rewording message to not say "request", which is a relatively technical term.

Reply:
ah I see. Nvm then. I'd rather we worded it differently, but no need to bother right now.